### PR TITLE
feat: Copy metadata and enhance copy for AWS promote quarantine

### PR DESCRIPTION
**Pull request by jack-c-tang**
# feat: Copy metadata and enhance copy for AWS promote quarantine

## Change Summary

* Copy valid metadata when copying files
* Apply S3 configuration suggested in the blog post about copying large files
https://aws.amazon.com/tw/blogs/storage/copying-objects-greater-than-5-gb-with-amazon-s3-batch-operations/

## PR Checklist

- [x] I've read and followed the [Contributing Guide](https://github.com/trendmicro/cloudone-filestorage-plugins/blob/master/.github/CONTRIBUTING.md).
- Documents/Readmes
  - [ ] Updated accordingly
  - [x] Not required
- Plugins that have versioning
  - [x] Version bumped and follows [Semantic Versioning](https://semver.org/)
  - [ ] Not required

### CODE REVIEW

File: post-scan-actions/aws-python-promote-or-quarantine/handler.py [hunk 1]
```diff
@@ -57,7 +57,9 @@
     CODE_MISC: 'incomplete scan due to miscellaneous reason. Provide the fss-scan-detail-code tag value to Trend Micro support',
 }
 
-s3 = boto3.client('s3')
+transfer_config = TransferConfig(max_concurrency=940, multipart_chunksize=16777216)
```
Comment (Laurence-Cheng) [inline post-scan-actions/aws-python-promote-or-quarantine/handler.py:60]: How about adding a constant for 16MB chunk size?

**Review by dougc-trend — DISMISSED** [review]
LGTM

**Review by Laurence-Cheng — APPROVED** [review]
LGTM

**Review by trend-tim-ling — DISMISSED** [review]

**Comment by jack-c-tang** [comment]
Fixed an issue when metadata cannot be retrieved.
https://github.com/trendmicro/cloudone-filestorage-plugins/pull/80/files#diff-6cad5d9dc88c1e7f608e9a5d3ba739020073f986af54e4ad284e34156ad05ee1R116

**Review by Laurence-Cheng — APPROVED** [review]

**Pull request merged**